enhancements/audit-command.md

```diff
@@ -0,0 +1,594 @@
+
+However, its re-usage shows allow us to deliver this proposal and requirement faster and with less effort. The main downside in using SDK binary might only the dependency required which will requires that users who do not use this tool needs to install it, but it can also be mitigated by providing a helper that will install the SDK binary locally when it is not found. For example, via Makefile target which would allow they customize the version of sdk that should be used.
+
+**Conclusion**
+//TODO: Add the consensus reached out. 
+
+3. How to extract the database from the index image without `oc` tool?
+
+## Summary
+
+This proposal describes a new operation provided via an alpha command/solution which has the purpose to audit all bundles of some specific OLM index catalog and then, output a report.
+
+## Motivation
+
+### Goals
+
+- Be able to audit all bundles of an OLM index catalog and output a report
+- Be able to easily check and identify aspects of the operators bundles which will be published on it such as:
+    - check if they are using deprecated apis
+    - check if they are built with sdk and its versions
+    - the language used 
+    - check if they are multi-arch
+    - check if they are using some relevant feature provided by looking their annotations
+    - check if they are following the recommendations and conventions defined
+    - check all warnings and errors which will be raised by using the SDK features scorecard and bundle validate
+- Be able to extract a report with the audit results and in some formats such as json. 
+- Be able to run specific custom checks for each bundle in order to audit the catalog
+
+### Non-Goals
+
+- Add new checks or validators. This proposal has the goal to define the command and operation only. The checks should be addressed in their respective proposals.
+- Allow users select what specific validators will be used by the SDK commands. However, it can be addressed in the future.  
+ 
+## Proposal
+
+Add a new command `--audit` with the purpose to audit all desired operator bundle data of a specific index catalog. 
+
+By default, the command output a report where it is possible to have a view of the main characteristics and aspect of all operator bundles shipped in a specified OLM index catalog as check if they are passing in criteria and checks required to ensure their quality. However, accordingly with the purpose of its usage is also possible customize what should or not be done by this operation. 
+
+This command will perform checks by using the [SDK](https://github.com/operator-framework/operator-sdk) available sub commands; `bundle validate` and `scorecard bundle`.
+
+In this way, opm binary will provide the alpha audit sub-command which would like: 
+
+```
+$ alpha --audit --help
```

Comment:
yep. It can be.  I did not want to add opm on that because we are not sure if it will or not be part of opm. See the open questions. 

I will add your suggestion as an example OK.